keps/sig-autoscaling/2702-graduate-v2beta2-autoscaling-api-to-GA/kep.yaml

```diff
@@ -0,0 +1,24 @@
+title: Graduate v2beta2 Autoscaling API to GA
+kep-number: 2702
+authors:
+  - "@supriya-premkumar"
+  - "@josephburnett"
+owning-sig: sig-autoscaling
+reviewers:
+  - "@josephburnett"
+  - "@gjtempleton"
+approvers:
+  - "@josephburnett"
+  - "@gjtempleton"
+prr-approvers:
```

Comment:
you don't need this here anymore (sorry bad template). just having it in the other file is enough